### PR TITLE
update Terraform and AKS readme

### LINKED ISSUE #623: TF: Document Terraform updates
Add documentation for the different updates made to terraform and infrastructure setup.

### PR DESCRIPTION
**Pull request by chenmliu**
# [Issue-id](https://github.com/retaildevcrews/ngsa/issues/623)

## PR Checklist

---

- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Testing

Closes https://github.com/retaildevcrews/ngsa/issues/623

### CODE REVIEW

**Comment by kev-ms** [comment]
Looks great. Would it be better to create separate markdown for customization and troubleshooting, and link them to terraform readme to keep deployment walkthrough more concise?

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md [hunk 1]
```diff
@@ -141,6 +141,52 @@ az aks update -g $aksClusterResourceGroupName -n $aksClusterName --aad-admin-gro
...
+
+## Customizations
+
+Some updates to the existing CAF (Cloud Adoption Framework) [repo](https://github.com/Azure/caf-terraform-landingzones-starter) that we made to meet the requiremnts for NGSA are:
+
+1. Change the deployment region to minimize latency
+
+The region was updated to `eastus2` to have the resources deployed in the U.S. region for better latency, instead of the Asia region, where the repo was originally developed in.
+
+Specifically, the `region1` variable in `..\enterprise_scale\construction_sets\aks\online\aks_secure_baseline\configuration\global_settings.tfvars` file was changed to `eastu2` and you can provide additional optional regions there.
```
Comment (AAkindele) [inline enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md]:
```suggestion
Specifically, the `region1` variable in `..\enterprise_scale\construction_sets\aks\online\aks_secure_baseline\configuration\global_settings.tfvars` file was changed to `eastus2` and you can provide additional optional regions there.
```

**Comment by chenmliu** [comment]
> Looks great. Would it be better to create separate markdown for customization and troubleshooting, and link them to terraform readme to keep deployment walkthrough more concise?

Moved to `enterprise_scale\construction_sets\aks\online\aks_secure_baseline\README.md`

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md (hunk 1 above)
Comment (chenmliu) [reply enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md]: Corrected typo

**Review by gortegaMS — APPROVED** [review]
Looks good to me

**Pull request merged**